src/dal-plugin/Stream.mm

```diff
@@ -125,8 +143,12 @@ - (CMSimpleQueueRef)copyBufferQueueWithAlteredProc:(CMIODeviceStreamQueueAltered
 }
 
 - (CVPixelBufferRef)createPixelBufferWithTestAnimation {
-    int width = 1280;
-    int height = 720;
+    
+    if (NSEqualSizes(self.testCardSize, NSZeroSize)){
```

Comment:
If you change the `testCardImageSize` method to just be `testCardSize`, then you can remove this `if` check and it should just work